examples/seismic/acoustic/wavesolver.py

```diff
@@ -211,10 +225,17 @@ def born(self, dmin, src=None, rec=None, u=None, U=None, m=None, **kwargs):
                          save=False, time_order=self.time_order,
                          space_order=self.space_order, dtype=self.model.dtype)
 
+        kwargs.update({'src': src})
+        kwargs.update({'rec': rec})
+        kwargs.update({'u': u})
+        kwargs.update({'U': U})
+        kwargs.update({'m': self.model.m})
```

Comment:
@mlange05 any thoughts on this kwargs updates? Shouldn't we at least avoid changing this parameter in-place ?